test/loader/test_link_neighbor_loader.py

```diff
@@ -194,18 +194,26 @@ def test_temporal_heterogeneous_link_neighbor_loader():
     data['author', 'paper'].edge_index = get_edge_index(200, 100, 1000)
 
     with pytest.raises(ValueError, match=r"'edge_label_time' was not set.*"):
-        loader = LinkNeighborLoader(data, num_neighbors=[-1] * 2,
-                                    edge_label_index=('paper', 'paper'),
-                                    batch_size=32, time_attr='time')
+        loader = LinkNeighborLoader(
+            data,
+            num_neighbors=[-1] * 2,
+            edge_label_index=('paper', 'paper'),
+            batch_size=32,
+            time_attr='time',
+        )
 
     # With edge_time:
     edge_time = torch.arange(data['paper', 'paper'].edge_index.size(1))
     paper_time_original = data['paper'].time.clone()
-    loader = LinkNeighborLoader(data, num_neighbors=[-1] * 2,
-                                edge_label_index=('paper', 'paper'),
-                                edge_label_time=edge_time, batch_size=32,
-                                time_attr='time', neg_sampling_ratio=0.5,
-                                num_workers=2)
+    loader = LinkNeighborLoader(
+        data,
+        num_neighbors=[-1] * 2,
+        edge_label_index=('paper', 'paper'),
+        edge_label_time=edge_time,
+        batch_size=32,
+        time_attr='time',
+        neg_sampling_ratio=0.5,
+    )
     for batch in loader:
```

Comment:
suprised that the `author_min>=0` test is passing, as all authors have a negative timestamp.

Reply:
It looks like only `paper` nodes have negative timestamp.